Exercise_2.java

```diff
@@ -7,21 +8,44 @@
        pivot and all greater elements to right 
        of pivot */
     void swap(int arr[],int i,int j){
-        //Your code here   
+        //Your code here
+        arr[i] = arr[i] + arr[j];
+        arr[j] = arr[i] - arr[j];
+        arr[i] = arr[i] - arr[j];
     }

```

Comment:
In place swap, should be giving you issues in some of test cases. Especially when an element is getting swapped by itself. Run a a few test cases